src/helpers/utilities.ts

```diff
@@ -177,7 +177,7 @@ export class Utilities {
    * Utility to check if the code is running inside of an add-in.
    */
   static get isAddin() {
-    return Utilities.host !== HostType.WEB;
+    return !!Utilities.host && Utilities.host !== HostType.WEB;
```

Comment:
FWIW, I think the bigger issue is that `Utilities.host` returns undefined.  This must be a regression (relative to the `"@microsoft/office-js-helpers": "0.7.4"` that Script Lab uses).

I remember running into something similar with Script Lab, hence we just stayed on 0.7.4.  It would be good to diff it to see what the real culprit is.

Let me see if I can identify the issue later today.

Reply:
FYI, I just submitted https://github.com/OfficeDev/office-js-helpers/pull/135
I would recommend that as a fix